Logging.Tests/LoggingTest.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Linq;
+using log4net.Config;
+using Logging.Entities;
+using NUnit.Framework;
+
+namespace Logging.Tests
+{
+    public class LoggingTest
+    {
+        private const string Hostname = "mockedHostname";
+        private const string Ip = "1.2.3.4";
+        private const string HttpRequestMethod = "POST";
+        private const string Path = "1.2.3.4/testRoute";
+        private const string Original = "Nunit Framework";
+        private const long Bytes = 42;
+        private const int StatusCode = 200;
+        private const decimal ResponseTime = 1337;
+        private const int ReqId = 1;
+        private const int Level = 30;
+        private static readonly long Time = DateTimeOffset.Now.ToUnixTimeMilliseconds();
+
+        [Test]
+        public void TestRequestLog()
+        {
+            var expectedLog =
+                $@"{{""level"":{Level},""time"":{Time},""reqId"":{ReqId},""http"":{{""request"":{{""method"":""{HttpRequestMethod}"",""userAgent"":{{""original"":""{Original}""}}}},""response"":{{""statusCode"":{StatusCode},""body"":{{""bytes"":{Bytes}}}}}}},""url"":{{""path"":""{Path}""}},""host"":{{""hostname"":""{Hostname}"",""ip"":""{Ip}""}},""responseTime"":{ResponseTime}.0}}";
+            var mockRequest = BuildCompletedRequestLog();
+
+            var appender = new log4net.Appender.MemoryAppender();
+            BasicConfigurator.Configure(appender);
+            LoggingUtility.LogRequest(mockRequest);
+
+            var result = appender.GetEvents();
+            Assert.IsTrue(result.Any());
```

Comment:
@TurtleARM all the tests under `Logging.Tests' were being ignored due to a missing test adapter. I included the adapter and ran all tests, now this test is failing on this assertion but I can't figure out how the appender should receive events.
![image](https://user-images.githubusercontent.com/34598738/94163864-a413f480-fe88-11ea-9218-e555dcf06796.png)

Reply:
Looks like MemoryAppender has some problems with github Actions, I changed the test logic to verify that the logging functions get called with the right parameters